int32 exposure_time
---
int32 error